controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -1022,6 +1022,10 @@ bool master_thread::handle_cmdu_1905_client_capability_report_message(
               << ", Result Code= " << result_code
               << ", client MAC= " << network_utils::mac_to_string(client_info_tlv->client_mac())
               << ", BSSID= " << network_utils::mac_to_string(client_info_tlv->bssid());
+
+    LOG_IF(!result_code.compare("SUCCESS"), DEBUG)
```

Comment:
(nitpick) `result_code == "SUCCESS"` is more natural to me.

Or actually, `client_capability_report_tlv->result_code() == wfa_map::tlvClientCapabilityReport::SUCCESS` is even better.

Reply:
Shouldn't really be a nitpick - its in our CONTRIBUTING.md AFAIR.